plugin/customcertificate/resources/css/form.css

```diff
@@ -3,16 +3,36 @@
     background-image: none;
     border: 1px solid #ccc;
     border-radius: 4px;
-    box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
     color: #555;
     font-size: 14px;
     line-height: 1.42857143;
     padding: 6px 12px;
-    transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
+    transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
 }
+
 .certificado-text-label {
     font-weight: 700;
     margin: 0 10px 0 10px;
 }
-input:disabled { background: #eee; }
-label { font-size: 13px; }
\ No newline at end of file
+
+input:disabled {
+    background: #eee;
+}
+
+label {
+    font-size: 13px;
+}
+
+.list-tags {
+    display: block;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.list-tags li {
+    display: inline-block;
+    margin-right: 10px;
+    margin-top: 5px;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)